templates/vue/src/components/Lightbox/media/MultiContentMedia/PageMenu/index.vue

```diff
@@ -35,16 +35,39 @@
           },
         ]"
       >
-        <ul class="page-menu-item fa-ul">
-          <page-menu-item
-            v-for="row in rows"
-            :key="row.node.id"
-            :node="row.node"
-            :lockRows="lockRows"
-            :disabled="disabledRow(row.node)"
-            @scroll-to="scrollToRef"
-          />
-        </ul>
+        <div
+          v-for="(menu, index) in filteredMenuGroups"
+          :key="index"
+          style="padding-top: 4px; padding-bottom: 4px;"
+        >
+          <b-card>

```

Comment:
I'm not sure what this component's template would look like. The majority of this PR's code is located in PageMenu/index.vue, which makes sense to me. It could be possible to create a new component that combines the PageMenu and PageRows components, but I imagine it would take a lot of refactoring, which might be better in a separate PR.

Reply:
Sorry I didn't mean to combine components. I rather meant to create a new component for menu groups that you have here so you can just loop over the menu groups and all the logic for those would go into that component (so basically each of these `b-card`s that you have here would be a MenuGroup component).

I also noticed that here you are putting the `v-for` on a div that includes just a `b-card` component inside. I don't see why you need the `div` -- you can just add the `v-for` right on the `b-card`.

Reply:
I've created a new MenuGroups.vue component. As for the `div`, I have tried moving the `v-for` and the other attributes to the `b-card`, but for some reason it will break the styling.